EasyPost.Tests/ServicesTests/CarrierAccountServiceTest.cs

```diff
@@ -60,6 +60,7 @@ public async Task TestCreateWithAlternateEndpoint()
             {
                 // the data we're sending is invalid, we want to check that the API error is because of malformed data and not due to the endpoint
                 Assert.Equal(400, e.StatusCode);  // 400 bad request is fine. We don't want a 404 not found
+                Assert.Equal("Malformed request. Please check the contents and retry.", e.Message);
```

Comment:
Mmm something is wrong here. It should be a 422 that returns this as a part of the error: `{"field": "account_number", "message": "must be present and a string"}`.

Please see the Python PR so we can get it here (I assume you need to pass `registration_data` as an empty object and this will clear up.

Reply:
Resolved